docs/cmdlets-reference/find-cohesityfilesnapshot.md

```diff
@@ -0,0 +1,126 @@
+# Find-CohesityFileSnapshot
+
+## SYNOPSIS
+Get the information about snapshots that contain the specified file or folder. In addition, information about the file or folder is provided.
+
+## SYNTAX
+
+```
+Find-CohesityFileSnapshot [-ClusterId <Int64>] [-ClusterIncarnationId <Int64>] [-FileName <String>] [-JobId <Int64>]
+ [-SourceId <Int64>] [<CommonParameters>]
+```
+
+## EXAMPLES
+
+### EXAMPLE 1
+```
+Find-CohesityFileSnapshot -FileName "abc.txt" -SourceId 123 -JobId 11
+```
+
+Returns snapshot information of specified file/folder in metioned source.
+
+
+### EXAMPLE 3
+```
+Find-CohesityFilesForRestore -Search "*txt" -Paginate $true -PaginationCookie rrrgcrsgre -PageSize 200
+```
+
+Returns next set of entries that match the search pattern "txt" based on provided Pagiantion cookie.
+
+## PARAMETERS
+
+### -ClusterId
```

Comment:
clusterId and incarnationId should not be part of the parameter, this is a cluster cmdlet